### DISCUSSION TITLE
this repository

### DISCUSSION THREAD

**Discussion by AntoniusGolly** [eugene-manuilov/typeorm-uml#52] [General]
Why are you creating a test repo without the main package installed? Don't let the users install the package on the test repo. It should work out of the box.

**Comment by eugene-manuilov** [comment]
What do you mean? Can you point me to a test repo?

**Reply by AntoniusGolly** [reply]
literally, this repository. like https://github.com/eugene-manuilov/typeorm-uml

It lacks the opportunity to generate a diagram with two commands. I would expect:
Run: 
```
npm i
npm some-command-to-generate-diagram-that-works-cross-platform
```
Not installing packages, or doing anything else.

**Reply by eugene-manuilov** [reply]
If I understand you correctly, then what you want is to be able to use this tool without adding it to your projects, right? You can do it by installing it globally with `npm i -g typeorm-uml` and then running `typeorm-uml /path/to/your/ormconfig.js`. Does it make sense to you?

**Reply by eugene-manuilov** [reply]
If I understand you correctly, then what you want is to be able to use this tool without adding it to your projects, right? You can do it by installing it globally with `npm i -g typeorm-uml` and then running `typeorm-uml /path/to/your/ormconfig.js`. Does it make sense to you?

**Reply by AntoniusGolly** [reply]
I'm fine with adding it to my project. I just think the repository should be functioning as minimal example rather than a how-to-do-these-steps in your repo. E.g. adding to project by `npm i typeorm-uml` is mandatory to do on my project, but on this repo it souldn't be necessary. 

Of course, I'm fine to adapt the ormconfig. 

Think of this flow: user is interested in your package and wants to experiment with it, see it in action. All he wants to do is
* clone repo
* adapt ormconfig.json (not create it, adapt it!)
* run `npm i`
* run a command that produces diagram

Also, the commands do not work out of the box on Windows. Instead of 

`$ ts-node ./node_modules/.bin/typeorm-uml ormconfig.json`
only this worked
`$ ts-node ./node_modules/typeorm-uml/bin/run ormconfig.json`
